packages/opds-web-client/src/components/context/ActionsContext.tsx

```diff
@@ -5,16 +5,25 @@ import DataFetcher from "../../DataFetcher";
 import { adapter } from "../../OPDSDataAdapter";
 
 // The main context for this app's Actions.
-export const ActionsContext = React.createContext<ActionsCreator | undefined>(
+type ActionsContextType =
+  | { fetcher: DataFetcher; actions: ActionsCreator }
```

Comment:
Do you need this first pipe?

Reply:
Yea that's how prettier formats it when you have a multi-line union. Makes it easier to read especially when you get over two unions.